dist/api_data.py

```diff
@@ -1605,6 +1605,10 @@ def __ge__(self, other):
 ]),
 
 'WT_SESSION.begin_transaction' : Method([
+    Config('evict_skip', 'false', r'''
+        if true this thread will not be used to help with eviction before or during this
```

Comment:
I assume we don't want "before or", or is there time travel involved? :-)

Reply:
It is accurate. In `begin_transaction` just before we set `WT_TXN_RUNNING` it calls `wt_cache_eviction_check`. So threads can be pulled into helping with eviction before the transaction starts. Similarly on the commit and rollback, there is a call just after `wt_txn_release`.

The "during" part is the less common path because if one begins a transaction and then lets it sit idle we also call `wt_txn_idle_cache_check`. So the setting on `begin_transaction` will also disable that call during the transaction.

Reply:
At the risk of splitting semantic hairs...

I would think that a typical reader/programmer just thinks of the transaction as starting as soon as their program calls `begin_transaction()`, and isn't aware of the finer distinction of where we check for eviction compared to where we set `WT_TXN_RUNNING`. So from that perspective it is all "during" the transaction.

Reply:
Good point. I'll get rid of "before"